browser/ui/views/omnibox/omnibox_autocomplete_browsertest.cc

```diff
@@ -59,12 +60,12 @@ IN_PROC_BROWSER_TEST_F(OmniboxAutocompleteTest, AutocompleteDisabledTest) {
 }
 
 IN_PROC_BROWSER_TEST_F(OmniboxAutocompleteTest, TopSiteSuggestionsEnabledTest) {
-  EXPECT_TRUE(browser()->profile()->GetPrefs()->GetBoolean(
-      kTopSiteSuggestionsEnabled));
+  EXPECT_TRUE(
```

Comment:
How about using unittest for checking default value instead of browser test?
It would be much better to use unittest if possible because browser test is much time/resource consuming.
I think it can be checked simply by each provider's unittests?

Reply:
Yeah, I didn't create these tests. I'll update that to be a unittest.

Reply:
To be honest, I'm not really sure it's that useful to test the default value of a pref - the only time the test is likely to fail is if we change the default value which is probably deliberate. I'm happy to just remove this test.

Reply:
Oops, I thought these tests were added in this PR. Sorry.
Regarding to the test itself, I think it's worth for having default value check tests because it could detect accidently changing. Yeah, most of time it'll be done intentionaly but who knows? :)
I think it's fine to leave as-is as it's not related your PR but please don't delete test code.

Reply:
I've already migrated them to a unittest, might as well leave it if that's all right with you?

Reply:
Oh, yes. Thanks! 👍🏼